integration_tests/Makefile

```diff
@@ -23,6 +23,19 @@ test-src: check
 
 	@echo "... Passed"
 
+.PHONY: test-src-no-isolation
+test-src-no-isolation: check
+	@echo "... Running integration tests for building src with --no-isolation --skip-dependency-check"
+
+	# Ensure the necessary build dependencies are installed in the
+	# host environment as we need to do so ourselves.
+
+	pip install meson-python meson cython
```

Comment:
How about installing these dependencies in a virtualenv to prevent polluting the host environment?

Reply:
🤔 I think I can't activate a virtual environment inside a Makefile for some reason. Maybe that's not possible. I'll look for a workaround.